src/main/java/com/iota/iri/network/pipeline/TransactionProcessingPipelineImpl.java

```diff
@@ -69,6 +70,12 @@
     private BlockingQueue<ProcessingContext> broadcastStageQueue = new ArrayBlockingQueue<>(100);
     private BlockingQueue<ProcessingContext> replyStageQueue = new ArrayBlockingQueue<>(100);
 
+    private int getNumberOfThreads() {
+        Reflections reflections = new Reflections(this.getClass().getPackage().getName());
+        //exclude the hashing stage as it is only run manually
+        return reflections.getSubTypesOf(com.iota.iri.network.pipeline.Stage.class).size() - 1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'com.iota.iri.network.pipeline.Stage' due to existing same package import 'com.iota.iri.network.pipeline.*'](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4627178)